### PR TITLE
Comments likes

### PR DESCRIPTION
**Pull request by MauricioRobayo**

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  has_many :likes, as: :likeable
```
Comment (Oghenebrume50) [inline app/models/comment.rb]: Are we liking comments? :thinking: there is no such functionality in your view and it isn't really required except of course you want to add it

**Comment by oscarnava** [comment]
Sorry, my bad! :-) 
I completely missed that, but it's solved now.

File: app/models/comment.rb (hunk 1 above)
Comment (oscarnava) [reply app/models/comment.rb]: Solved!

Comment (MauricioRobayo) [reply app/models/comment.rb]: You're totally right, that was our idea, but we got into the requirements and forgot that we were planning to do that :D. Thanks for bringing that up!

**Pull request merged**